vignettes/email-overview.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Überblick Open-Access-Metadaten-Schnelltest"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Überblick Open-Access-Metadaten-Schnelltest}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Überblick Open-Access-Metadaten-Schnelltest
+===========================================
+
+Überblick
+---------
+
+Umfangreiche Metadaten sichern die Auffindbarkeit und Nachnutzung von
+Open-Access-Artikeln. Aus diesem Grunde ist die Überprüfung der
+Metadaten ein wesentlicher Workflow-Bestandteil beim Umgang mit
+Open-Access-Publikationskosten und Transformationsverträgen.
+
+Der Open-Access-Metadaten-Schnelltest ermöglicht die automatische
+Überprüfung der Metadatenqualität von Open-Access-Zeitschriftenartikeln.
+Er richtet sich in erster Linie an Bibliotheken und wissenschaftliche
+Informationseinrichtungen, die Open-Access-Publikationskosten etwa im
+Rahmen eines Publikationsfonds oder eines Transformationsvertrags
+verwalten.
+
+Im Mittelpunkt der Überprüfung des  Open-Access-Metadaten-Schnelltests 
+stehen die Metadaten der DOI-Registrierungsagentur Crossref, welche von den 
+meisten wissenschaftlichen Verlagen als Metadatenservice genutzt wird. Daher
+greifen vielfältige Forschungsvorhaben, Discovery- und
+Data-Analytics-Serivces auf die Crossref-Datendienste zu, um
+Open-Access-Artikel zu finden und nachzunutzen.
+
+Testergebnisse
+--------------
+
+Das Testergebnis des Open-Access-Metadaten-Schnelltest wird per Email
+verschickt und beinhaltet folgende Ergebnisse:
+
+### Indexierungsstatus in Crossref
+
+Der Abschnitt fasst zusammen, für welche DOIs keine Metadaten in
+Crossref verfügbar sind. Es kann vielfältige Ursachen haben, warum eine DOI
+nicht in Crossref indexiert ist:
+
+-   DOI ist syntaktisch fehlerhaft
+
```

Comment:
these empty lines are not recommended in markdown style guides, and they can cause a fair bit of ambivalence.

Depending on the markdown parser, these lines will either:
- wrap the content in a separate `<p>` tag, and it's not clear whether that is sound HTML
- create separate lists for each element (which is clearly not intended here)
- treat the whole thing as a normal list

To avoid this ambivalence, it's best not to add extra linebreaks.

will fix in commit

Reply:
done